src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java

```diff
@@ -134,7 +134,16 @@ protected boolean submit(O op) throws InterruptRunException, IllegalStateExcepti
                         // if(success) return true;
                         break;
                     case READ:
-                        // if(success) return true;
+                    	final String path = pathOp.dstPath();
+						final String scopeName = path.substring(0,path.indexOf("/"));
+						final String streamName = path.substring(path.indexOf("/")+1, path.length());
+						if (scopeMap.get(scopeName).get(streamName)==null) {
```

Comment:
Do we need to check this on READ operation? If we can't read from this path should we to create it? To my mind, it is applicable only for CREATE/UPDATE operations and throw exception/error/NULL in case we can't read.

Reply:
You're right. We should not create any scope/stream if we are trying to read. The app should set the corresponing status ("not found") and log error message in that case